Sources/PrivoSDK/model/RegisterResponse.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+struct RegisterResponse: Codable {
+    let to: To
+
+    struct To: Codable {
```

Comment:
why do we need an embedded struct here?

Reply:
this allows to use auto-synthesized compiler tricks for Codable and have less and clearer code than if were overriding writing parsing manually